### ISSUE TITLE
FlairContact - A simple way for teams to have shared contact lists

### ISSUE THREAD

**Issue by omranjamal** [DevArifHossain/saas-idea-validator#5]
# FlairContact

## Description
A simple and easy way for teams and organizations to have shared contact lists. Imagine an organization sharing their key contacts with the entire organization and it being available instantly on everyone's actual phone's contact list, or a sales team having a shared contact list so as to identify callers easily. This will make collaboration and on-boarding new employees easy.

## Use Cases
- Teams sharing contact information with each-other easily without asking in O(n^2) complexity
- Organizations sharing key contacts with everyone in their organization
- Teams adding and sharing external contacts (like client numbers) with other members of their sales team.
- Families sharing key contacts with everyone
- Individuals sharing contacts with each other in a way their contact never becomes invalid once they change their number.

**Comment by DevArifHossain** [comment]
Can't this be done by just creating a notion page where everyone fills out their contact info?

**Comment by omranjamal** [comment]
True! Fair point. I feel there might be a valid use case for the sales team, because the idea came about when my co-founder and I didn't have a shared contact list for big clients and all the investors we were meeting; but I guess that can be solved with Notion too.
